src/xcode/ENA/ENAUITests/ENAUITests-Extensions.swift

```diff
@@ -35,6 +35,13 @@ extension XCUIElement {
 		guard exists, !frame.isEmpty else { return false }
 		return XCUIApplication().windows.element(boundBy: 0).frame.contains(frame)
 	}
+	
+	func waitAndTap(_ timeout: Double = .short) {
+		// Need XCTAssertEqual() here because we can append a message, this is not possible for XCTAssertTrue()
+		// swiftlint:disable:next xct_specific_matcher
+		XCTAssertEqual(self.waitForExistence(timeout: timeout), true, "Could not find XCUIElement: \(self) before tapping it.")
```

Comment:
`isHittable` might prevent this:

```suggestion
		XCTAssertTrue(self.waitForExistence(timeout: timeout))
		XCTAssertTrue(self.isHittable)		
```

> isHittable returns true if the element exists and can be clicked, tapped, or pressed at its current location. It returns false if the element does not exist, is offscreen, or is covered by another element.
>
> **Note**
isHittable only returns true if the element is already visible and hittable onscreen. It returns false for an offscreen element in a scrollable view, even if the element would be scrolled into a hittable position by calling click(), tap(), or another hit-point-related interaction method.

([Apple](https://developer.apple.com/documentation/xctest/xcuielement/1500561-ishittable))

Reply:
Hmm yeah, but isHittable would only add here another check, but it does not replaces another check....?
But I'd love to let here the error message, because we use this generic approach for so many calls (< 340), that could make it more easy to get for wich element it does not work.